I had to add npm to dev dependencies

Strange issue here. I have npm installed globally, but when I followed your setup instructions, it was choking on the lambda_package task with:

    Loading "lambda_package.js" tasks...ERROR
    >> Error: Cannot find module 'npm'

I added npm to the dev dependencies inside your project, and everything ran fine. Not sure why that is if you didn't encounter the same issue. Again, I have npm installed globally, so it seems like a strange error.

Hi @adamrneary and @taphat, sorry for the delay in following this up.

Would you mind putting the following in a file like test.js outside your project folder then run `node test.js`:

```
var npm = require("npm");

console.log(npm);
```

Do you get `Cannot find module 'npm'` when running that? For me I get a print out of the object but I suspect the npm module might not be installed globally for everyone.

FYI the packaging tests pass on travis without explicitly including the npm module in devDependencies but npm can be a bit finicky about include paths.

I'm thinking I might either be able to add npm explicitly to the grunt-aws-lambda dependencies to ensure it's included for everyone, or add a warning if the module can't be found so people know to add it to their devDependencies.

I got this error too.

    module.js:340
        throw err;
          ^
    Error: Cannot find module 'npm'
        at Function.Module._resolveFilename (module.js:338:15)
        at Function.Module._load (module.js:280:25)
        at Module.require (module.js:364:17)
        at require (module.js:380:17)

--- a/package.json
+++ b/package.json
@@ -39,7 +39,8 @@
         "grunt-contrib-clean": "^0.5.0",
         "grunt-contrib-nodeunit": "^0.3.3",
         "grunt": "~0.4.5",
-        "adm-zip": "~0.4.4"
+        "adm-zip": "~0.4.4",
+        "npm": "^2.10.0"
     },
     "peerDependencies": {
         "grunt": "~0.4.5"

